pkg/kubectl/bind-apiservice/plugin/servicebindings_test.go

```diff
@@ -0,0 +1,824 @@
+/*
```

Comment:
would love to see us merge `servicebindings.go` and this test, plus the API types first, to get this in step by step instead of 6000 lines at once.